package/doc/sphinx/source/documentation_pages/selections.rst

```diff
@@ -300,13 +300,31 @@ bynum *index-range*
     selects atoms 5 through 10 inclusive. All atoms in the
     :class:`MDAnalysis.Universe` are consecutively numbered, and the index
     runs from 1 up to the total number of atoms.
-
+    
+id *index-range*
+    selects all atoms in the given range which consists of two numbers 
+    separated by a colon such as ``id 5:10`` (both inclusive).
+    e.g. ``id 1`` selects the atom with id 1; ``id 5:10`` selects the
+    atoms with ids 5 through 10 both inclusive. All atoms in the
+    :class:`MDAnalysis.Universe` are consecutively numbered, though
+    id (an topology attribute) can be labeled by the user and need 
+    not be continuous. Conventionally it corresponds to the serial number 
+    in PDB format.
+     
 index *index-range*
     selects all atoms within a range of (0-based) inclusive indices,
     e.g. ``index 0`` selects the first atom in the universe; ``index 5:10``
     selects atoms 6 through 11 inclusive. All atoms in the
     :class:`MDAnalysis.Universe` are consecutively numbered, and the index
     runs from 0 up to the total number of atoms - 1.
+    
+    
+.. note::
+    While ``index`` is the underlying position of an atom in a linear array
```

Comment:
I am not convinced that the line "is the underlying position of an atom in a linear array of atoms in the universe" is a good fit here. The description is a bit abstract and the user often prefers practical information. I think the note should briefly say the difference between id and index in a way that is practical from the user's perspective.